src/units.js

```diff
@@ -1,17 +1,18 @@
-/* eslint no-console:0 */
+// @flow
 
 /**
  * This file does conversion between units.  In particular, it provides
  * calculateSize to convert other units into ems.
  */
 
 import ParseError from "./ParseError";
+import Options from "./Options";
 
 // This table gives the number of TeX pts in one of each *absolute* TeX unit.
 // Thus, multiplying a length by this number converts the length from units
 // into pts.  Dividing the result by ptPerEm gives the number of ems
 // *assuming* a font size of ptPerEm (normal size, normal style).
-const ptPerUnit = {
+const ptPerUnit: {[string]: number} = {
```

Comment:
Is this necessary?

Reply:
No; Flow will infer the type automatically. Ditto for line 34 as well.

Reply:
hm, on second thought, Flow isn't as good at inferring types when you use computed properties, so I think it's probably good to keep this annotation and the one below.

Reply:
Removed this and the one on line 34.